tests/service/server_flask_test.py

```diff
@@ -1,11 +1,12 @@
 # pylint: disable=no-self-use,invalid-name
 import json
 
-from allennlp.service.server_flask import app
+from allennlp.service.server_flask import make_app
 from allennlp.service.servable import ServableCollection
 from allennlp.common.testing import AllenNlpTestCase
 
 # Add default models to app
+app = make_app()
```

Comment:
Can we put this in `setUp`, for both of these, so that it's not run when pytest scans the tests?